### ISSUE TITLE
Set wwactive doesn't work

### ISSUE THREAD

**Issue by mafradon** [Victor-Mo/IRT-ESP#5]
Since yesterday i updated my interface with the latest version. I used to only use the set_water commando to start the boiler. This worked fine. But i cant seem tot set the tap water boiler on and off. What commando do u use to do that? 

I have tried wwactive , tapwater and wwonetime but all of Them have no effect to the wwheat parameter in the boiler data. Alsof the mqtt comands to do the same dont seem to work.

Hope you can help.

**Comment by Victor-Mo** [comment]
The 'boiler wwactive on' and 'boiler wwactive off' should switch the tap water on and off. It does seem to correctly set it on the boiler, but it is not reported in the status (info command).  I will have a look, because something is wrong.

**Comment by Victor-Mo** [comment]
@mafradon I have checked the tap water setting. The telnet command is 'boiler wwactive on/off'. The MQTT topic is 'boiler_cmd_wwactivated' (for example: 'home/irt-esp/boiler_cmd_wwactivated') a value of 0/1 will switch it on and off. 

The problem was reporting the tap water status. The 0x05 message was implemented, but it was not processed/called. I have fixed that. Just keep in mind it can take a minute before the status is reported. So if you switch the tap water it can take a minute before showing up.  It is currently building a new firmware. As added bonus this version also reports the outdoor temperature (if you have an outdoor sensor).
